airbyte-integrations/connectors/source-tiktok-marketing/acceptance-test-config.yml

```diff
@@ -31,11 +26,23 @@ tests:
 
   discovery:
     - config_path: "secrets/prod_config.json"
+      backward_compatibility_tests_config:
+        disable_for_version: "0.1.14"
```

Comment:
you're not changing the spec here, are you? if I'm not mistaken this config is used for spec backward compatibility tests

Reply:
oh that's discovery, nevermind